third-party/linkerd/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@kubernetes-models/linkerd",
+  "version": "0.0.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tommy351/kubernetes-models-ts.git"
+  },
+  "homepage": "https://github.com/tommy351/kubernetes-models-ts/tree/master/third-party/linkerd",
+  "author": "YC Kao <yckaokarl@gmail.com>",
+  "license": "MIT",
+  "main": "index.js",
+  "module": "index.mjs",
+  "types": "index.d.ts",
+  "sideEffects": false,
+  "scripts": {
+    "build": "ts-node scripts/download-crd.ts && crd-generate && publish-scripts build",
+    "prepack": "publish-scripts prepack"
+  },
+  "publishConfig": {
+    "access": "public",
+    "directory": "dist",
+    "linkDirectory": true
+  },
+  "keywords": [
+    "kubernetes",
+    "kubernetes-models",
+    "linkerd"
+  ],
+  "engines": {
+    "node": ">=14"
+  },
+  "dependencies": {
+    "@kubernetes-models/apimachinery": "workspace:^",
+    "@kubernetes-models/base": "workspace:^",
+    "@kubernetes-models/validate": "workspace:^",
+    "tslib": "^2.4.0"
+  },
+  "devDependencies": {
+    "@kubernetes-models/crd-generate": "workspace:^",
+    "@kubernetes-models/publish-scripts": "workspace:^",
+    "@kubernetes-models/read-input": "workspace:^",
+    "@types/js-yaml": "^4.0.5",
+    "@types/node": "^18.7.23",
+    "execa": "^5.1.1",
+    "js-yaml": "^4.1.0",
+    "ts-node": "^10.9.1",
+    "typescript": "^4.8.4",
+    "vitest": "^0.29.8"
+  },
+  "crd-generate": {
+    "input": [
+      "./crds/crd.yaml"
```

Comment:
Maybe you can try the files in [this folder](https://github.com/linkerd/linkerd2/tree/main/charts/linkerd-crds/templates) instead?

Reply:
Files in this folder will contain helm template like
`{{ include "partials.annotations.created-by" . }}` or `helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}` so we cannot use it directly.
In this PR I'm following the official [docs](https://linkerd.io/2.14/tasks/install-helm/#linkerd-crds) step with default to generate the crds.